internal/uplink/gpustats.go

```diff
@@ -3,22 +3,41 @@ package uplink
 // TODO: change to "/gs-api/gpu-stats"
 const GPUStatsUrl = "/gs-api/status/"
 
-type StatsPackage struct {
-	Hostname string   `json:"hostname"`
-	Stats    GPUStats `json:"stats"`
+type GpuStatsUpload struct {
+	Hostname string          `json:"hostname"`
```

Comment:
NIT: Now that we don't share these with the frontend, we could just remove these json tags.

Reply:
Maybe, let's see how the refactors actually impact the front-end first.